queries/private-cloud/helpers.ts

```diff
@@ -100,6 +100,15 @@ export function getPrivateCloudProjectsQuery({
       },
     ];
   }
+  if (cluster) {
+    if (Array.isArray(cluster)) {
+      // Handle the case where cluster is an array
```

Comment:
A comment is necessary only if the code lacks `self-documentation`. I think, `Array.isArray` is sufficiently intuitive on its own.